recipes/highs/all/conanfile.py

```diff
@@ -65,7 +65,7 @@ def build(self):
         apply_conandata_patches(self)
         cmake = CMake(self)
         cmake.configure()
-        cmake.build(target="libhighs")
+        cmake.build(target="highs")
```

Comment:
I'm guessing the target changed between versions? If the old target is correct for the old one, this should be conditional on the current version, no?

Reply:
Yes, target name changed. The new name also works with the old version :-)

Reply:
Oh, that's great then, thanks for clarifying!